portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -35,23 +36,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         Query query = new Query("Comment-" + page);
 
         if (!text.equals("undefined")) {
-            Filter textPropertyFilter = new FilterPredicate("text", FilterOperator.EQUAL, text);
-            Filter authorPropertyFilter = new FilterPredicate("author", FilterOperator.EQUAL, author);
-            Filter emotionPropertyFilter = new FilterPredicate("emotion", FilterOperator.EQUAL, emotion);
-            CompositeFilter filter =  CompositeFilterOperator.and(textPropertyFilter, authorPropertyFilter, emotionPropertyFilter);
+            Filter textPropertyFilter = new FilterPredicate("text",
+                FilterOperator.EQUAL, text);
+            Filter authorPropertyFilter = new FilterPredicate("author",
+                FilterOperator.EQUAL, author);
+            Filter emotionPropertyFilter = new FilterPredicate("emotion",
+                FilterOperator.EQUAL, emotion);
+            CompositeFilter filter = CompositeFilterOperator
+                .and(textPropertyFilter, authorPropertyFilter,
+                    emotionPropertyFilter);
             query.setFilter(filter);
-        } 
+        }
 
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        DatastoreService datastore = DatastoreServiceFactory
+            .getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
 
-        List<Key> keys = new ArrayList<>();
-        for (Entity entity : results.asIterable()) {
+        List < Key > keys = new ArrayList < > ();
```

Comment:
Nit: Weird bracket spaces here too. Is auto-format doing this?

Reply:
Yeah, seems like that